google-cloud-datastore/src/main/java/com/google/cloud/datastore/AggregationResult.java

```diff
@@ -38,7 +41,19 @@ public AggregationResult(Map<String, LongValue> properties) {
    * @return An aggregation result value for the given alias.
    */
   public Long get(String alias) {
```

Comment:
I would also add getLong, and we can encourage users to use the explicit type

Reply:
sure, do you think it would be good idea to mark older method `get` with @Deprecated annotation ?

Reply:
fixed as part of 403c01c.